icons/llvm.svg

```diff
@@ -1 +1 @@
084-.184-.7979.012-.9419.18-.175.204-.209.464-.14.608.021.045.07.082.138.11.002.221.137.75.404 1.1339a4.9636 4.9636 0 01-.108-.667c-.009-.16 0-.274.015-.35.047.328.174.684.577 1.125-.248-.54-.22-.862-.195-1.144l.009-.044.006-.001c.036.263.142.534.482.964a.412.412 0 01-.014-.028l.014.021a1.6259 1.6259 0 01-.167-1.03l.008-.003c.046.03.091.07.134.126.128.172.188.349.212.47a.528.528 0 00.067.384c-.245.179-.485.346-.72.501-.23.149-.49.313-.757.47-.14.0799-.279.1579-.414.2299l-.1349-.077h.002l-.008-.004c-.97-.559-1.6899-.932-2.5178-1.082-.931-.1669-1.7868.283-1.9828 1.082a1.2839 1.2839 0 00.552 1.3799l.012.008c.013.01.026.017.04.026l.06.036h.002c.1919.11.4239.191.6909.237l.028.005h.006a3.4997 3.4997 0 00.412.044c.9748.047 2.1557-.408 3.3226-.982.1.054.202.108.306.166 2.5468 1.403 4.0977 1.5849 6.1535.728-.502.151-1.2079.356-2.0938.344a.05.05 0 00-.02-.006l.001.006a5.4605 5.4605 0 01-.648-.05v-.001l-.03-.003v-.003a.267.267 0 00-.001.002 6.2495 6.2495 0 01-1.5558-.436 7.7924 7.7924 0 01-.745-.362 57.0652 57.0652 0 01-1.0059-.569c.348-.178.692-.363 1.026-.5449.3549-.189.6719-.362.9179-.5l.009-.004.007-.001s.083-.018.16-.042c.17-.052.4099-.149.6949-.283.028.117.095.215.195.253.235.091.407.027.522-.102l.018-.01.047-.037c.05-.03.107-.067.156-.09a1.14 1.14 0 01.0529-.022c.04-.01.078-.023.116-.037l.033-.01c-.145.297-.091.581-.091.581.043-.371.22-.608.392-.715.064-.027.13-.05.2-.067.064 0 .116.023.144.07.07.12.217.194.364.226.024.15-.045.404-.4.866l.09-.062.004-.003.072-.053c.143-.108.2519-.21.3339-.305a.97.97 0 00.191-.305c.012.2-.043.507-.311 1.036 0 0 .317-.276.514-.63.006.191-.01.443-.063.785.345-.673.325-1.085.239-1.326l-.01-.036a.433.433 0 00-.01-.065l.042-.057c.118-.166-.048-.3439-.154-.4149-.06-.04-.2-.154-.454-.157-.225-.27-.51-.15-.958.145a1.2969 1.2969 0 01-.166.093.725.725 0 00-.055-.117c.16-.096.326-.199.494-.308.317-.207.553-.416.728-.615.51-.5169.69-1.0198 1.2099-1.4458l-.013.001.06-.032c.854-.458 1.3069-.547 1.3389-.553l-.003.128.018-.125a1.7949 1.7949 0 01.481.344 2.1198 2.1198 0 01.362.487l.044.301s0-.117-.01-.33c.1529-.152 1.0198-1.044 1.7238-2.4668.694-1.4039.9439-2.5228 1.0169-2.9238a18.1885 18.1885 0 01.665 2.4658s-.002-.133-.031-.374c.097-.336.5159-1.8718.6269-3.7556.1-1.6809-.065-2.7798-.14-3.1738.648.537.9949 1.307.9949 1.307s-.04-.163-.188-.437l.02.027s-.236-.455-.644-.905a6.4495 6.4495 0 00-.157-.172c-.4319-.4589-.8539-.6169-1.1698-.6719-.1-.02-.203-.03-.308-.03h-.019c-.125-.003-.201.002-.207-.026-.02-.095.11-.285.11-.285s-.177.046-.554.33c-.23.167-.5199.41-.8529.76a13.6889 13.6889 0 00-.8899.9949l-.075.095a.203.203 0 00-.023-.017c.076-.331.362-1.118 1.7879-3.1118zM11.92 14.0908a.351.351 0 01.012.006zm-.263 2.6058c.088.044.19.106.303.194h-.005a.484.484 0 00-.155.07.956.956 0 01-.126-.224zm-4.8546 1.5699c.132 0 .276.014.432.048l.02.005.017.004c.11.027.223.062.337.102l.057.021c.191.074.395.156.611.248.3199.139.6189.2769.8378.3749l.183.088c-.221.106-.424.196-.577.261-1.0138.416-1.8298.536-2.3377.391-.945-.269-.76-1.536.42-1.542z"/></svg>
```

Comment:
If possible, could you get this down to 3 decimals? No worries if not, the path for this is ***RIDICULOUS***!

Reply:
@adamrusted Done 🐉 🐲